src/hooks/useStakingValidator.ts

```diff
@@ -42,9 +49,54 @@ export const useStakingValidator = () => {
 
     const response = await getValidators();
 
-    const filteredValidators = response?.validators.filter((validator) =>
+    // Filter out jailed and unbonded validators
+    const availableValidators =
+      response?.validators.filter(
+        (validator: Validator) => validator.status === 3 && validator.jailed === false
```

Comment:
[nit] extract 3 to a const somewhere `UNBONDED_VALIDATOR_STATUS`

Reply:
You can import:
`import { BondStatus } from '@dydxprotocol/v4-client-js/build/node_modules/@dydxprotocol/v4-proto/src/codegen/cosmos/staking/v1beta1/staking';
`
and then do:
`BondStatus.BOND_STATUS_BONDED`

Reply:
ah good point, should probably just be importing from `cosmos/staking/v1beta1/staking`